change/@fluentui-react-badge-1d02e00e-35d1-4d9a-8f0e-9a522c35bd8d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26655&commitId=957ff181c4aef460a4966ba7d9238e8b4ac51c69&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1830 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|PresenceBadge Converged - OOF status.default - RTL.chromium.png|31| Changed|
|PresenceBadge Converged - OOF status.default.chromium.png|31| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>